.github/workflows/upload_latest_files.yml

```diff
@@ -12,6 +12,12 @@ jobs:
     name: Upload
     runs-on: ubuntu-latest
     steps:
+      - name: Set up Go 1.14
+        uses: action/setup-go@v1
```

Comment:
I think the v2 version can be used, which can be more stable.

Reply:
Got it!